docs/get-started/vue3-vite.md

```diff
@@ -277,14 +277,35 @@ The definition above will generate the following controls:
 
 ### Limitations
 
-`vue-component-meta` cannot currently reference types from an import alias. You will need to replace any aliased imports with relative ones, as in the example below. See [this issue](https://github.com/vuejs/language-tools/issues/3896) for more information.
+If you're working with a project that relies on [`tsconfig references`](https://www.typescriptlang.org/docs/handbook/project-references.html) to link to other existing configuration files (e.g. `tsconfig.app.json`, `tsconfig.node.json`), we recommend that you update your [`.storybook/main.js|ts`](../configure/index.md) configuration file and add the following:
 
 ```ts
-// YourComponent.ts
-import type { MyProps } from '@/types'; // ❌ Cannot be resolved
-import type { MyProps } from '../types'; // ✅ Can be resolved
+// .storybook/main.ts
+import type { StorybookConfig } from '@storybook/vue3-vite';
+
+const config: StorybookConfig = {
+  framework: {
+    name: '@storybook/vue3-vite',
+    options: {
+      docgen: {
+        plugin: 'vue-component-meta',
+        tsconfig: 'tsconfig.app.json',
+      },
+    },
+  },
+};
+
+export default config;
 ```
 
+<Callout variant="info">
+
+This is not a limitation of Storybook but instead from the `vue-component-meta` package. For more information, refer to the appropriate [GitHub issue](https://github.com/vuejs/language-tools/issues/3896).
+
```

Comment:
I suggest slightly changed to "This is not a limitation of Storybook but rather a behavior stemming from the `vue-component-meta` package. For further details, please consult the relevant [GitHub issue](https://github.com/vuejs/language-tools/issues/3896)."

Reply:
@chakAs3, @larsrickert, are you both ok with the following wording inside the callout:

"This is not a limitation of Storybook, but instead how `vue-component-meta` works. For more information, refer to the appropriate [GitHub issue](https://github.com/vuejs/language-tools/issues/3896)."

The rationale is to keep the wording simple and accessible to everyone without introducing any misconceptions. As this will probably be the default behavior for the foreseeable future, we're also safeguarding ourselves from any potential questions or issues that might arise. Leaving it to the reader to decide to check the issue to learn more. Wouldn't you agree?

Reply:
@jonniebigodes Sounds good, I changed it like that :) Should we also change the headline? "Limitations" might also give the wrong impression here. Maybe something like "Configuration" or `tsconfig references`?

Reply:
Agree on the heading, and what about we change it to "Override the default configuration"? Will that work for you?

Reply:
@jonniebigodes  it is not an "Override" rather than it is an explicit supply for a valid tsconfig that contains files path

Reply:
@chakAs3 , as a user, I could perceive it as such since the default pattern is the following:
```ts
import type { StorybookConfig } from '@storybook/vue3-vite';

const config: StorybookConfig = {
  framework: {
    name: '@storybook/vue3-vite',
    options: {
      docgen: 'vue-component-meta',
    },
  },
};

And in this case, we're overriding the default behavior/pattern by adding the `tsconfig` configuration element. Wouldn't you agree?

Reply:
@jonniebigodes yes you are right from out side it is an "override", However for vue-component-meta it is not. we have to be explicit about the tsconfig that we pass to the checker  
so go ahead it is just a detail, i believe that with this doc user has enough information if he is troubleshooting

Reply:
@jonniebigodes Done :)

Reply:
Appreciate it, @larsrickert. I've approved it. @chakAs3 I understand where you're coming from. The suggestion was more based on the overall narrative of the documentation. We can always get back to it and adjust it, more even when the comment from @baixiaoyu2997 is properly vetted, and we adjust it accordingly.